internal/command/views/operation.go

```diff
@@ -115,12 +115,12 @@ func (v *OperationHuman) Plan(plan *plans.Plan, schemas *terraform.Schemas) {
 	}
 
 	// Side load some data that we can't extract from the JSON plan.
-	var opts []jsonformat.PlanRendererOpt
+	var opts []plans.Quality
 	if !plan.CanApply() {
-		opts = append(opts, jsonformat.CanNotApply)
+		opts = append(opts, plans.NoChanges)
```

Comment:
Just sanity checking here: the new name captures all the reasons why `!plan.CanApply()` ? The old name seems to fit this description better since plan.CanApply is a black box

Reply:
🤔 That's a good question — I was going from the perspective of the output we traditionally give, and from that view, "CanNotApply" isn't really the truth — you can totally apply an empty plan. 

And the logic inside CanApply() comes down to "are there concrete changes? OR, is it refresh-only and there's refresh changes?" 

To me, yes, that sounds like they're equivalent... and if they later change to Not be equivalent, well, it's all in the `internal` packages and we can rationalize distinctions as they arise.